triage/templates/triage/wizard-step-summary.html

```diff
@@ -19,6 +19,10 @@ <h3 class="title">About you</h3>
 				<dt class="answer">Do you use online marketplaces to sell your products?</dt>
 				<dd class="answer" id="used_online_marketplace">{{ all_cleaned_data.used_online_marketplace|yesno:"Yes,No" }}</dd>
 			{% endif %}
+            {% if all_cleaned_data.is_in_companies_house is not None %}
```

Comment:
test would be nice :P

Reply:
We don't have any test for templates. I'll add a ticket to add them